lib/app.js

```diff
@@ -320,6 +324,7 @@ window.GraphJS = GraphJSScripts;
 
 window.GraphJS = {
 	init,
+	reMountRiotTags,
```

Comment:
are you kidding me? This would break backwards compatibility!!

Reply:
Why will it?? 🤔 This is just a new function into GraphJS.. Not touching any of other functions or tags.. 

This will be like an additional feature for react-graphjs..

Reply:
ok can you name it properly. It should be "remountRiotTags"

Reply:
changed it